engine/docs-runner/src/main/scala/org/enso/docsrunner/Main.scala

```diff
@@ -0,0 +1,50 @@
+package org.enso.docsrunner
+
+import java.io.File
+import org.enso.interpreter.runtime.{Context => EnsoContext}
+import org.enso.languageserver.io.{
+  ObservableOutputStream,
+  ObservablePipedInputStream
+}
+import org.enso.polyglot.{LanguageInfo, MethodNames}
+import org.graalvm.polyglot.Context
+
+object Main {
```

Comment:
This project and module should not exist at all. Integrate your new command into the engine-runner main. Also talk to @radeusgd about how to integrate it into our launcher.